libsolidity/codegen/ir/IRGenerator.cpp

```diff
@@ -73,10 +76,21 @@ pair<string, string> IRGenerator::run(ContractDefinition const& _contract)
 	return {warning + ir, warning + asmStack.print()};
 }
 
-string IRGenerator::generate(ContractDefinition const& _contract)
+string IRGenerator::generate(
+	ContractDefinition const& _contract,
+	map<ContractDefinition const*, string const&> const& _otherYulSources
+)
 {
 	solUnimplementedAssert(!_contract.isLibrary(), "Libraries not yet implemented.");
 
+	auto subObjectSources = [&_otherYulSources](std::set<ContractDefinition const*> const& subObjects) -> string
+	{
+		std::string subObjectsSources;
+		for (ContractDefinition const* subObject: subObjects)
```

Comment:
This creates nondeterminism.

Reply:
maybe to use `fullyQualifiedName` in `_otherYulsources`, and to make `m_subObjects` to be set of string, in that case, contracts will be sorted by qualified name. Sounds good?

Reply:
`std::set<VariableDeclaration const*, ASTNode::CompareByID>`